app/main.py

```diff
@@ -1,8 +1,30 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        # when obj created add link on it to the dict "people"
```

Comment:
Please pay attention to the checklist, there is no need to add comments to the code as it is clear and self-explanatory.